### PR TITLE
Add pause changefeed recommendation for upgrade

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-10270

This PR adds a pause changefeed job recommendation to the upgrade docs in v23.1 (i.e., upgrade to v23.1). In earlier versions of CRDB (v23.1 and lower), changefeed jobs will get slower the more nodes move over to the newer version during a rolling upgrade. 

If I understand correctly, the behavior that slows down changefeed jobs in 23.1 and lower upgrades has been resolved, so that v23.1 -> v23.2 upgrades should not encounter it. However, **question: do we still want to include a more general recommendation to pause running changefeeds during upgrades in later version upgrades as well?** We do briefly mention it on this page https://www.cockroachlabs.com/docs/stable/changefeed-messages#node-restarts re node restarts and duplicate messages — but this should be in the upgrade docs, if it is a more general rec.

For rendered previews, refer to the **files changed** box below.

### CODE REVIEW

**Comment by kathancox** [comment]
cc @nicktrav

**Comment by rharding6373** [comment]
> question: do we still want to include a more general recommendation to pause running changefeeds during upgrades in later version upgrades as well?

No, we do test version upgrades and haven't encountered any performance issues that would benefit from pausing changefeeds when upgrading from v23.1+. I think that a general recommendation is unnecessary at this point.

**Review by rharding6373 — APPROVED** [review]
LGTM Thank you!

**Comment by kathancox** [comment]
> > question: do we still want to include a more general recommendation to pause running changefeeds during upgrades in later version upgrades as well?
> 
> No, we do test version upgrades and haven't encountered any performance issues that would benefit from pausing changefeeds when upgrading from v23.1+. I think that a general recommendation is unnecessary at this point.

Thanks for confirming @rharding6373 !

File: src/current/v23.1/upgrade-cockroach-version.md [hunk 1]
```diff
@@ -312,6 +334,27 @@ Finalization is required only when upgrading from {{ previous_version }}.x to {{
...
     If the cluster continues to report that it is on the previous version, finalization has not completed. If auto-finalization is enabled but finalization has not completed, check for the existence of [decommissioning nodes]({% link {{ page.version.version }}/node-shutdown.md %}?filters=decommission#status-change) where decommission has stalled. In most cases, issuing the `decommission` command again resolves the issue. If you have trouble upgrading, [contact Support](https://cockroachlabs.com/support/hc/).
 
+1. If you paused [changefeed]({% link {{ page.version.version }}/change-data-capture-overview.md %}) jobs before starting the upgrade process, you can now [resume]({% link {{ page.version.version }}/resume-job.md %}) the jobs:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    RESUME JOB {changefeed_job_ID};
+    ~~~
+
+    Or, pause all changefeed jobs:
```
Comment (rmloveland) [inline src/current/v23.1/upgrade-cockroach-version.md:344]: "or, resume" ?

**Review by rmloveland — APPROVED** [review]
LGTM modulo one nit

File: src/current/v23.1/upgrade-cockroach-version.md (hunk 1 above)
Comment (kathancox) [inline src/current/v23.1/upgrade-cockroach-version.md:344]:
```suggestion
    Or, resume all changefeed jobs:
```

Comment (kathancox) [reply src/current/v23.1/upgrade-cockroach-version.md]: Oooops, yes! Thanks for catching that!

**Pull request merged**